casper/src/test/scala/coop/rchain/casper/MergeabilityRules.scala

```diff
@@ -328,6 +331,39 @@ trait MergeabilityRules {
 
   }
 
+  /***
+   There was a COMM within one of the deploys caused on one side by a peek.
+   The other deploy had an event without a match in TS, of same polarity to the data matched by peek.
+   These could not spawn more work.
+    */
+  object PeekedNoMatch extends MergeableCase {
+
+    override def precondition(
```

Comment:
code duplication alert

Reply:
Thanks for noticing this, I need to make preconditions a bit more sophisticated